2022-round-1/XiaobenYin/nx_version.txt

```diff
@@ -0,0 +1 @@
+2.6.3
```

Comment:
thank @XiaobenYin !

Could you update the version of networkx, the latest one is 2.7.1
Please do also update the version of python on you machine :)

Reply:
Thank you @MridulS !

I've updated the version of Python and networkX on my laptop, rerun the test and updated the results in the folder. The version of networkX of mine is 2.8rc2.dev0, I wonder if that's acceptable?

Reply:
Thanks! No problem :)